### PR TITLE
Add cookbook image demos, fix dismissable link

### PR DESCRIPTION
**Pull request by Cretezy**
I added images/GIFs for almost all of the cookbook recipes.

The GIFs for the `fading-in-images` was larger than the others (4MB) since it has an actual image, and GIF compression is very bad on real images. There's 2x ~1.5MB and the rest of the other GIFs are usually under 0.5MB.

I also renamed the dismissable from dismissible, as it was a type. I couldn't find where to add a redirect.

Also fixed a small mistake in the README about linkcheck, from `--input` to `--input-file`.

### CODE REVIEW

**Comment by Cretezy** [comment]
I signed it!

File: cookbook/gestures/dismissable.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
 title: "Implement Swipe to Dismiss"
-permalink: /cookbook/gestures/dismissible/
+permalink: /cookbook/gestures/dismissable/
```
Comment (brianegan) [inline cookbook/gestures/dismissable.md]: Could we please keep this as `dismissible`? I thought it was `dismissable` as well, haha, but it turns out that's actually not a word!  😄 

https://docs.flutter.io/flutter/widgets/Dismissible-class.html

File: cookbook/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ reference to help you build up an application.
 
   * [Handling Taps](/cookbook/gestures/handling-taps/)
   * [Adding Material Touch ripples](/cookbook/gestures/ripples/)
-  * [Implement Swipe to Dismiss](/cookbook/gestures/dismissible/)
+  * [Implement Swipe to Dismiss](/cookbook/gestures/dismissable/)
```
Comment (brianegan) [inline cookbook/index.md]: `dismissible`

File: cookbook/gestures/dismissable.md [hunk 3]
```diff
@@ -159,3 +159,5 @@ class MyApp extends StatelessWidget {
   }
 }
 ```
+
+![Dismissable Demo](/images/cookbook/dismissable.gif)
```
Comment (brianegan) [inline cookbook/gestures/dismissable.md]: dismissible.gif

**Comment by Cretezy** [comment]
Should be good to go!

**Comment by brianegan** [comment]
Thanks again for the contribution! LGTM, just need some help from @mjohnsullivan or @filiph for a final review / merge.

File: cookbook/gestures/dismissible.md [hunk 4]
```diff
@@ -9,13 +9,13 @@ we're writing an email app, we might want to allow our users to swipe away email
...
 the Trash.
 
-Flutter makes this task easy by providing the [`Dismissable`](https://docs.flutter.io/flutter/widgets/Dismissible-class.html)
+Flutter makes this task easy by providing the [`Dismissible`](https://docs.flutter.io/flutter/widgets/Dismissible-class.html)
 Widget.
 
 ## Directions
 
   1. Create List of Items
-  2. Wrap each item in a `Dismissable` Widget
```
Comment (brianegan) [inline cookbook/gestures/dismissible.md]: Haha, I thought I had fixed all of these! 😭

Comment (Cretezy) [reply cookbook/gestures/dismissible.md]: I fixed all the ones I found grep!

**Comment by filiph** [comment]
This is awesome, thank you @Cretezy! Merging.

**Pull request merged**